examples/java/rest/api/v2010/account/CallCreator.java

```diff
@@ -57,25 +57,26 @@
 
 
 public class CallCreator extends Creator<Call>{
-    private String AccountSid;
     private String RequiredStringProperty;
+    private String AccountSid;
     private List<String> TestArrayOfStrings;
+
+    public CallCreator(final String RequiredStringProperty) {
+        this.RequiredStringProperty = RequiredStringProperty;
+    }
     public CallCreator(final String AccountSid, final String RequiredStringProperty) {
-    
         this.AccountSid = AccountSid;
-    
         this.RequiredStringProperty = RequiredStringProperty;
-    
     }
 
-    public CallCreator setAccountSid(final String AccountSid){
-        this.AccountSid = AccountSid;
-        return this;
-    }
     public CallCreator setRequiredStringProperty(final String RequiredStringProperty){
         this.RequiredStringProperty = RequiredStringProperty;
         return this;
     }
+    public CallCreator setAccountSid(final String AccountSid){
+        this.AccountSid = AccountSid;
+        return this;
+    }
```

Comment:
Hrm, why is there a setter for the account SID here but not in `CallDeleter`? Note that the current helper doesn't have a setter: https://github.com/twilio/twilio-java/blob/main/src/main/java/com/twilio/rest/api/v2010/account/CallCreator.java